src/fedora/29/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM fedora:29
```

Comment:
I'm curious why this Dockerfile isn't based on the image produced from the `src/fedora/29/Dockerfile`?  Are there scenarios in which the dependencies will diverge?  I naively would think we would want to automatically get any changes the product teams make in `src/fedora/29/Dockerfile`.

Reply:
No good reasons, wasn't sure how to do that.